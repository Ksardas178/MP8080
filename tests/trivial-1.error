line 3: <WARNING> argument adapted to match 2 8-bit value(s)
line 4: <WARNING> argument adapted to match 2 8-bit value(s)
line 4: <WARNING> argument adapted to match 2 8-bit value(s)
