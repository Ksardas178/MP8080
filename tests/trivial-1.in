MOV C H
MOV 46
MOV 57
MOV hf