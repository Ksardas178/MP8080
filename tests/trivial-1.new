initialized
initialized
line 2: <ERROR> wrong register name recieved
line 3: <WARNING> argument adapted to match 3 8-bit value(s)
line 3: <ERROR> expected 3 argument(s)
line 2: get cmd with 3 args (1, 4, 607105823)
initialized
line 4: <WARNING> argument adapted to match 3 8-bit value(s)
line 4: <ERROR> wrong register name recieved
line 4: <WARNING> argument adapted to match 3 8-bit value(s)
line 4: <ERROR> expected 3 argument(s)

[30;1mCode analysis results:
[0m______________________________
00000000: 00000001
00000001: 00000001
00000010: 00000100
00000011: 00000001
00000100: 00000111
00000101: 00000001
