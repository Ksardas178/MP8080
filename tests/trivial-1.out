
[31;1mErrors: 2
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: MOV 1 4
